File: Course.py
class Course(object):
    def __init__(self):
        self.code = ""
        self.name = ""
        self.AU = 0
        self.preReq = []
        self.description = ""

class Schedule(object):
    def __init__(self):
        self.code = ""
        self.index = ""
        self.group = ""
        self.type = ""
        self.day = ""
        self.time = ""
        self.venue = ""
        self.remark = ""

File: models/course.go
package models

type Course struct {
	Code        string `json:"code" db:"code"`
	Name        string `json:"name" db:"name"`
	AU          int    `json:"AU" db:"au"`
	PreReq      string `json:"preReq" db:"preReq"`
	Description string `json:"description" db:"description"`
}
